### ISSUE TITLE
Log  dir is empty.

### ISSUE THREAD

**Issue by xfdywy** [ShangtongZhang/DeepRL#10]
Hi, I just run your code and I can see the output in the terminal. But the log dir is empty. I don't know how to output the log info into the log file. Besides, how can I use the tensorboard? Looking forward to  your reply. Thanks.

**Comment by ShangtongZhang** [comment]
It's amazing that your name is the same as my ex girlfriend....
By default log dir is empty, I didn't log anything. I assume you know how to use tensorboard in tensorflow. If it's not the case, you need to learn that first. Then you can use `config.logger.xxx` to log anything you want, you can find xxx in `utils/tf_logger.py`

**Comment by xfdywy** [comment]
...............It's really amazing. And I am sure I am not your ex girlfriend....
Logger is a small problem.
Besides, in the atari_warpper, the ClippedRewardsWrapper class seems to use np.sign, is that correct? I saw other implementations that use something like min(max(reward,-1) , 1) .
Another thing. Because of that you wrapper the reward directly, the test reward is also -1,0,1 and it seems not convenient to see the true mean episodes reward which always reported in the papers.





 I am sure I am not your ex girlfriend because I am a boy, hhhhhhh.......

**Comment by ShangtongZhang** [comment]
`min(max(reward,-1) , 1)` is wrong, you can find training details in the nature paper. `np.sign` does make sense because it means the frequency of positive/negative rewards.
It's true that we cannot recover original rewards. This doesn't happen for later algorithms like D3PG, P3O. But for DQN and A3C, I forget to address this issue. It's easy to fix, you can just remove that wrapper and do same thing as I did in P3O. I'll patch it soon. Thanks for pointing it out.
